src/confluent_kafka/src/Admin.c

```diff
@@ -1053,6 +1048,14 @@ static PyObject *Admin_incremental_alter_configs(Handle *self,PyObject *args,PyO
                 return NULL; /* Exception raised by find() */
         }
 
+        ConfigEntry_type = cfl_PyObject_lookup("confluent_kafka.admin",
+                                                  "ConfigEntry");
+        if (!ConfigEntry_type) {
+                Py_DECREF(ConfigResource_type);
+                rd_kafka_AdminOptions_destroy(c_options);
+                return NULL; /* Exception raised by find() */
+        }
+
```

Comment:
Scope of `ConfigEntry` is in the function `Admin_incremental_config_to_c`. Why are we getting the type here?

Reply:
to avoid repeating the lookup N times

Reply:
Makes sense. Did realize this to be in a for loop.